nest-tax-backend/src/tasks/tasks.service.ts

```diff
@@ -187,6 +188,7 @@ export class TasksService {
     if (process.env.FEATURE_TOGGLE_REMINDER_UNPAID_TAX !== 'true') {
       return
     }
+    const TWENTY_DAYS_AGO = dayjs().subtract(20, 'day').toDate() // 20 days ago
```

Comment:
The comment here is unnecessary, as it is very clear form the variable name (and also quite readable from the code now too) what is the intention here.